tests/unit/e_db_mysqlTest.php

```diff
@@ -286,17 +286,18 @@ public function testUpdate()
 			$db->delete('tmp');
 
 			// Test 1
```

Comment:
Each test should be its own method.  That way, individual tests could be run to validate the behavior of one "unit".